modules/asset-manager-impl/src/test/java/org/opencastproject/assetmanager/impl/AssetManagerWithSecurityTest.java

```diff
@@ -168,13 +153,13 @@ private Object parametersForTestTakeSnapshot() {
     final Organization org = TestOrganization.mkDefault();
     return $a($a(acl(ace(ROLE_TEACHER, WRITE_ACTION)),
                  TestUser.mk(org, ROLE_USER),
-                 false),
+                 true),
```

Comment:
This should still be false?  Your ACL is ROLE_TEACHER, and you're trying to take the snapshot as ROLE_USER...